packages/account/src/Components/personal-details/personal-details.jsx

```diff
@@ -1,30 +1,31 @@
-import { Formik, Field } from 'formik';
-import React from 'react';
 import {
-    Modal,
-    Autocomplete,
     AutoHeightWrapper,
+    Autocomplete,
     Checkbox,
-    Dropdown,
-    DesktopWrapper,
-    MobileWrapper,
     DateOfBirthPicker,
+    DesktopWrapper,
     Div100vhContainer,
+    Dropdown,
     FormSubmitButton,
     Input,
+    MobileWrapper,
+    Modal,
     Popover,
     RadioGroup,
     SelectNative,
-    ThemedScrollbars,
     Text,
+    ThemedScrollbars,
 } from '@deriv/components';
+import { Field, Formik } from 'formik';
+import { Localize, localize } from '@deriv/translations';
+import { PlatformContext, getLegalEntityName, isDesktop, isMobile, routes, toMoment, validPhone } from '@deriv/shared';
+
+import FormSubHeader from '../form-sub-header';
 import { Link } from 'react-router-dom';
-import { localize, Localize } from '@deriv/translations';
-import { getLegalEntityName, isDesktop, isMobile, routes, toMoment, PlatformContext, validPhone } from '@deriv/shared';
+import React from 'react';
+import classNames from 'classnames';
```

Comment:
Can we follow the import rules ?
https://github.com/binary-com/deriv-app/blob/master/docs/JavaScript/README.md#import-rules

Reply:
Will refactor this